tests/test_states/test_ghz.py

```diff
@@ -44,23 +49,15 @@ def test_ghz_4_7():
     np.testing.assert_equal(np.all(bool_mat), True)
```

Comment:
```suggestion
    np.testing.assert_array_equal(res, expected_res)
```
Delete line 48 if you accept this suggestion. Github won't let me get rid of line 48 here because you did not make changes to the line in this diff.